test/e2e/auth/pod_security_policy.go

```diff
@@ -325,6 +329,9 @@ func restrictedPSP(name string) *policy.PodSecurityPolicy {
 			RunAsUser: policy.RunAsUserStrategyOptions{
 				Rule: policy.RunAsUserStrategyMustRunAsNonRoot,
 			},
+			RunAsGroup: &policy.RunAsGroupStrategyOptions{
+				Rule: policy.RunAsGroupStrategyMustRunAs,
+			},
```

Comment:
i think you would need to change the actual code that does the verification of  what the container is running as

Reply:
I Could not understand what changes you want to change because It is already specified here https://github.com/kubernetes/kubernetes/blob/5647244b0c13db98816c136ad3e7d58551bbd41d/pkg/apis/policy/validation/validation.go#L236

Reply:
my point was to verify whether the actual running container is indeed running as 999. I think we verify in other e2e. So here just verifying that PSP admission is working as expected is fine.